Source/HTTP/httpcall_request.cpp

```diff
@@ -100,6 +130,29 @@ HCHttpCallRequestSetRequestBodyString(
     );
 }
 
+STDAPI
+HCHttpCallRequestSetRequestBodyReadFunction(
+    _In_ HCCallHandle call,
+    _In_ HCHttpCallRequestBodyReadFunction readFunction,
+    _In_ uint32_t bodySize
```

Comment:
That's correct. I think this could be tweaked in the future to support chunked transfer encoding when the body size may not be known but shouldn't be the default because some endpoints don't seem to support or allow chunked requests. For example, while testing SISU was failing my requests with chunked transfer encoding but succeeded without it